meta-resin-common/recipes-containers/docker-disk/docker-resin-supervisor-disk/resin-supervisor.service

```diff
@@ -38,8 +36,8 @@ ExecStart=@BASE_BINDIR@/bash -c 'source @SBINDIR@/resin-vars && \
     -e LISTEN_PORT=$LISTEN_PORT \
     -e SUPERVISOR_IMAGE=${SUPERVISOR_IMAGE} \
     ${SUPERVISOR_IMAGE}'
-ExecStop=-@BINDIR@/docker stop resin_supervisor
-ExecStop=-@BINDIR@/docker rm --force resin_supervisor
+ExecStop=@BINDIR@/docker stop -t 2 resin_supervisor
```

Comment:
A small nit - would be nicer to have the long-form flag to make it clear what this parameter is and to be consistent with other docker commands, e.g. `--time 2`.